packages/cli/src/commands/deprecated/test.ts

```diff
@@ -35,15 +35,18 @@ const commandModule: CommandModule<Options, Options> = {
       ...(forgeOpts?.split(" ") || []),
     ]);
 
-    // Reset LibDeploy.sol
-    console.log("Reset LibDeploy.sol");
-    await resetLibDeploy(testDir);
-
-    process.on("SIGINT", () => {
+    process.on("SIGINT", async () => {
       console.log("\ngracefully shutting down from SIGINT (Crtl-C)");
+      await resetLibDeploy(testDir);
       child.kill();
```

Comment:
similarly, wondering if we should kill first, then reset
```suggestion
      child.kill();
      await resetLibDeploy(testDir);
```